File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_TTR_Update.ascx.cs
using NotificationSystem.NotificationSystem.Data.Classes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
	public partial class ctrSearch_TTR_Update : System.Web.UI.UserControl
	{
		protected void Page_Load(object sender, EventArgs e)
		{
			clsNotificationSystem theNotificationSystem = new clsNotificationSystem();
			if (!(Page.IsPostBack))
			{
				cmbTicket.DataSource = theNotificationSystem.GetTT();
				cmbTicket.DataTextField = "TroubleTicketNo"; 
				cmbTicket.DataValueField = "Status";
				cmbTicket.DataValueField = "Type";
				cmbTicket.DataBind();

			//	cmbType.DataSource = theNotificationSystem.GetTypeList();
			//	cmbType.DataTextField = "TroubleTicket";
			//	cmbType.DataValueField = "Type";
			//	cmbType.DataBind();
			}
		}

		public void ClearControl()
		{
			this.cmbTicket.Value = string.Empty; 
			cmbTicket.Text = "--";
			//this.cmbType.Value = string.Empty; 
			//cmbType.Text = "--";
		}
	}
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_TTR_Update.ascx.vb
Imports System
Imports System.Web.UI
Imports NotificationSystem.NotificationSystem.Data.Classes

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_TTR_Update
        Inherits UserControl
        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Dim theNotificationSystem As New clsNotificationSystem
            If Not Me.Page.IsPostBack Then
                cmbTicket.DataSource = theNotificationSystem.GetTT()
                cmbTicket.DataTextField = "TroubleTicketNo"
                cmbTicket.DataValueField = "Status"
                cmbTicket.DataValueField = "Type"
                cmbTicket.DataBind()

                '	cmbType.DataSource = theNotificationSystem.GetTypeList();
                '	cmbType.DataTextField = "TroubleTicket";
                '	cmbType.DataValueField = "Type";
                '	cmbType.DataBind();
            End If
        End Sub

        Public Sub ClearControl()
            cmbTicket.Value = String.Empty
            cmbTicket.Text = "--"
            'this.cmbType.Value = string.Empty; 
            'cmbType.Text = "--";
        End Sub
    End Class
End Namespace
